### PR TITLE
Add support for new `packages` field that is going to replace `storyPackages`

### PR DESCRIPTION
**Pull request by TBonnin**
# Add support for new `packages` field
## What does this change?

New story packages are now returned by CAPI into the `packages` field which is going to replace `storyPackages`
This change checks for this new `packages` field content and fall back over old `storyPackages` if empty.
## Does this affect other platforms - Amp, Apps, etc?

App team is making the same change at the moment. @davidfurey
## Screenshots

Before
screen shot 2016-02-25 at 16 10 13

After
screen shot 2016-02-25 at 16 10 00
## Request for comment

@rich-nguyen

### CODE REVIEW

File: common/app/model/RelatedContent.scala [hunk 1]
```diff
@@ -25,8 +25,17 @@ case class RelatedContent (
 
 object RelatedContent {
   def apply(parent: ContentType, response: ItemResponse): RelatedContent = {
-    // It's misleading to use storyPackage here rather than relatedContent. A tidy up should rename this file.
-    val items = response.storyPackage.map { item =>
+    // It's misleading to use storyPackage here rather than relatedContent. A tidy up should rename this object
```
Comment (TBonnin) [inline common/app/model/RelatedContent.scala]: I agree this is very misleading. It took me more than it should have to understand how `RelatedContent` works.
I am happy to spend some time and refractor this once this PR is out. @rich-nguyen

File: project/Dependencies.scala [hunk 2]
```diff
@@ -25,7 +25,7 @@ object Dependencies {
   val commonsHttpClient = "commons-httpclient" % "commons-httpclient" % "3.1"
   val commonsIo = "commons-io" % "commons-io" % "2.4"
   val cssParser = "net.sourceforge.cssparser" % "cssparser" % "0.9.18"
-  val contentApiClient = "com.gu" %% "content-api-client" % "7.27"
+  val contentApiClient = "com.gu" %% "content-api-client" % "7.28"
```
Comment (TBonnin) [inline project/Dependencies.scala]: Necessary to include packages related bugfix https://github.com/guardian/content-api-scala-client/blob/master/CHANGELOG.md#728

**Comment by rich-nguyen** [comment]
:+1: You may need to run tests again, because you've upgraded content-api.

File: common/app/model/RelatedContent.scala (hunk 1 above)
Comment (rich-nguyen) [inline common/app/model/RelatedContent.scala]: Yes, `RelatedContent`could do with another look, now that the content model is a bit more stable.

**Pull request merged**